### PR TITLE
Update Townsfolk.lua

### PR DESCRIPTION
**Pull request by anon1231823**

### CODE REVIEW

File: Localization/Translations/MinimapIcon/Townsfolk.lua [hunk 1]
```diff
@@ -252,19 +252,19 @@ local townsfolkLocales = {
...
         ["esES"] = "Objetos comerciables",
-        ["frFR"] = "Fournitures",
+        ["frFR"] = "Fournitures d'artisanat",
     },
     ["Vendor"] = {
-        ["ptBR"] = "Vendedor",
+        ["ptBR"] = "Comerciante",
         ["ruRU"] = "Торговец",
-        ["deDE"] = "Händler",
+        ["deDE"] = "Verkäufer",
```
Comment (BreakBB) [inline Localization/Translations/MinimapIcon/Townsfolk.lua:260]: "Händler" fits better for this translation.

```suggestion
        ["deDE"] = "Händler",
```

**Comment by anon1231823** [comment]
@BreakBB Actually if you look at the words used in the german localization, it's Verkäufer

See
https://de.classic.wowhead.com/npc=3323/horthus
https://de.classic.wowhead.com/npc=1257/keldric-boucher
https://de.classic.wowhead.com/npc=1285/thurman-mullby

File: Localization/Translations/MinimapIcon/Townsfolk.lua (hunk 1 above)
Comment (anon1231823) [reply Localization/Translations/MinimapIcon/Townsfolk.lua]: @BreakBB Actually if you look at the words used in the german localization, it's Verkäufer

See
https://de.classic.wowhead.com/npc=3323/horthus
https://de.classic.wowhead.com/npc=1257/keldric-boucher
https://de.classic.wowhead.com/npc=1285/thurman-mullby
https://de.classic.wowhead.com/npc=4585/ezekiel-graves
https://de.classic.wowhead.com/npc=3369/gotri

**Comment by BreakBB** [comment]
Hey @anon1231823 wow, that is really a thing 😅 As a German persona, playing on an English client I would totally expect it to be translated into "Händler" instead of "Verkäufer 🙈 

Will change it again, thanks for the hint and all the great PRs! 🙏🏻

**Pull request merged**